src/pydisagg/age_split/__init__.py

```diff
@@ -0,0 +1,4 @@
+# age_split/__init__.py
+from . import age_split, age_var, helper
```

Comment:
I think there is no need to export the modules, since syntax-wise, it will make no difference. You will still have to do
```
from age_split.age_split import ...
```
Unless you want to export anything (function, class) in `age_split.age_split` we can keep this empty.

Reply:
I was wondering about trying to change this file subdirectory to ihme in the future so that we can keep the Open Source and IHME versions in the same place but only have the user call the IHME functions if they need it. My current solution I dont think achieves that... something like

Reply:
But i think I will try that in the next update